### DISCUSSION TITLE
Not able to load template for validation

### DISCUSSION THREAD

**Discussion by harsh2403** [projectdiscovery/nuclei#3868] [Q&A]
![image](https://github.com/projectdiscovery/nuclei/assets/40105975/03e75e1d-fba7-4d7f-aee5-9f3a0191965a)

I have updated to the latest release as well

**Comment by ehsandeep** [comment]
@harsh2403, you can add `-v` or `-debug` to see the error in the template or share the template for further support.

**Reply by harsh2403** [reply]
![image](https://github.com/projectdiscovery/nuclei/assets/40105975/e4d8154b-1433-41ca-8608-d123b0705b8f)

**Reply by ehsandeep** [reply]
Yes, something is wrong with your template; you need to share the template for the help.

**Reply by harsh2403** [reply]
Thanks for the support its resolved.
